swap/src/protocol/alice/event_loop.rs

```diff
@@ -360,7 +361,14 @@ where
     fn new_handle(&mut self, peer: PeerId, swap_id: Uuid) -> EventLoopHandle {
         // we deliberately don't put timeouts on these channels because the swap always
         // races these futures against a timelock
-        let (transfer_proof_sender, mut transfer_proof_receiver) = bmrng::channel(1);
+
+        // TODO: I had to increase the channel buffer to get
+        //  concurrent_bobs_after_xmr_lock_proof_sent test passing.
+        //  I don't really understand why. In my understanding the channel created here
+        //  is for one specific swap.  Send a transfer proof through this channel
+        //  should still only happen once, as this is a specific channel for a specific
+        //  swap. How is this triggered multiple times?
+        let (transfer_proof_sender, mut transfer_proof_receiver) = bmrng::channel(10);
```

Comment:
It shouldn't be necessary, you are right. I wonder if this issue goes away if we implement it the way I suggest in https://github.com/comit-network/xmr-btc-swap/pull/405/.